src/inim.nim

```diff
@@ -248,10 +247,24 @@ proc runForever() =
             tempIndentCode = ""
             continue
 
+
         # Special commands
         if currentExpression in ["exit", "exit()", "quit", "quit()"]:
             cleanExit()
 
+        if currentExpression in ["help", "help()"]:
+            outputFg(fgCyan, true):
+                echo("""
+                iNim - Interactive Nim Shell - By AndreiRegiani
+
+                Available Commands:
+                Quit - exit, exit(), quit, quit()
+                Help - help, help()
+                """
```

Comment:
I think you need to de-indent this string, it looks like this when you print it:

```
nim> help 
                iNim - Interactive Nim Shell - By AndreiRegiani

                Available Commands:
                Quit - exit, exit(), quit, quit()
                Help - help, help()
```

try https://nim-lang.org/docs/strutils.html#unindent%2Cstring%2CNatural%2Cstring

Reply:
Otherwise looks good!